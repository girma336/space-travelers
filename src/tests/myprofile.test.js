import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import React from 'react';
import MyProfile from '../components/myprofile';
import store from '../Redux/configureStore';

describe('Test the react component of my profile', () => {
  it('Test react My Profile page', () => {
    const myProfile = render(
      <Provider store={store}>
        <MyProfile />
      </Provider>,
    );
    expect(myProfile).toMatchSnapshot();
  });
});
