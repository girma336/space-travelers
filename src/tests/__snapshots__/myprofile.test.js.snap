// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the react component of my profile Test react My Profile page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="container-mission"
        >
          <h1
            class="display-6"
          >
            My Rocket
          </h1>
          <div
            class="table-responsive-sm"
          >
            <table
              class="table table-bordered table-hover"
            >
              <thead>
                <tr>
                  <th>
                    Rocket Name
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    You have not reserved any rockets yet.
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div
          class="container-rocket"
        >
          <h1
            class="display-6"
          >
            My Missions
          </h1>
          <div
            class="table-responsive-sm"
          >
            <table
              class="table table-bordered table-hover"
            >
              <thead>
                <tr>
                  <th>
                    Mission Name
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    You have not joined any missions yet.
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="container-mission"
      >
        <h1
          class="display-6"
        >
          My Rocket
        </h1>
        <div
          class="table-responsive-sm"
        >
          <table
            class="table table-bordered table-hover"
          >
            <thead>
              <tr>
                <th>
                  Rocket Name
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  You have not reserved any rockets yet.
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
      <div
        class="container-rocket"
      >
        <h1
          class="display-6"
        >
          My Missions
        </h1>
        <div
          class="table-responsive-sm"
        >
          <table
            class="table table-bordered table-hover"
          >
            <thead>
              <tr>
                <th>
                  Mission Name
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  You have not joined any missions yet.
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
